commcare_export/env.py

```diff
@@ -388,6 +388,17 @@ def _or(*args):
         pass
 
 
+@unwrap('val')
+def substr(val, start, end):
+    if not val:
+        return None
+
+    if start < 0 or end < 0:
+        return None
+
+    return val[start:end]
```

Comment:
What happens here if `val` is not a string?

Reply:
Good point. That would fail. Based on the other functions, like 'selected', I decided to coerce the argument to unicode.

Reply:
nit: could add a test case for this